### PR TITLE
Fix Tendermint RPC Validator types

### LINKED ISSUE #659: Fix validator type in tendermint rpc
The output from rpc looks like:

`curl http://localhost:26658/validators | jq .result`

```json
{
  "block_height": "845",
  "validators": [
    {
      "address": "E359FA31E749FFDC571AA4E5F629F70CD53D328A",
      "pub_key": {
        "type": "tendermint/PubKeyEd25519",
        "value": "L6f41DzvG/B+jUIc+avA3d0+BOhqgKB0pfiYSaug+2I="
      },
      "voting_power": "3",
      "proposer_priority": "0"
    }
  ],
  "count": "1",
  "total": "1"
}
```

The [types from tendermint-rpc](https://github.com/CosmWasm/cosmjs/blob/52dc82bbe4799c2b4fcd1e69a81c9980ffcdc7c8/packages/tendermint-rpc/src/responses.ts#L141-L145) look like:

```ts
export interface ValidatorsResponse {
  readonly blockHeight: number;
  readonly results: readonly Validator[];
}

export interface Validator {
  readonly address?: Uint8Array;
  readonly pubkey: ValidatorPubkey;
  readonly votingPower: number;
  readonly accum?: number;
  readonly name?: string;
}
```

Please:
* (failing) test that calling validators gets a list of one result with pubkey and non-zero voting power.
* rename results -> validators
* remove accum/name and add proposerPower

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #659

### CODE REVIEW

File: packages/tendermint-rpc/src/client.spec.ts [hunk 1]
```diff
@@ -130,14 +130,31 @@ function defaultTestSuite(rpcFactory: () => RpcClient, adaptor: Adaptor, expecte
...
+    pendingWithoutTendermint();
+    const client = await Client.create(rpcFactory(), adaptor);
+    const response = await client.validators();
+
+    expect(response).toBeTruthy();
+    expect(response.blockHeight).toBeGreaterThanOrEqual(1);
+    expect(response.count).toBeGreaterThanOrEqual(1);
+    expect(response.total).toBeGreaterThanOrEqual(1);
+    expect(response.validators.length).toBeGreaterThanOrEqual(1);
+    expect(response.validators[0].address!.length).toEqual(20);
```
Comment (webmaster128) [inline packages/tendermint-rpc/src/client.spec.ts:143]: Any clue if/why this address can be unset? Would be good to have a note in `export interface Validator {` if you do

**Review by webmaster128 — APPROVED** [review]

File: packages/tendermint-rpc/src/client.spec.ts (hunk 1 above)
Comment (willclarktech) [reply packages/tendermint-rpc/src/client.spec.ts:143]: Checking the type from the codec it doesn't seem to be optional.

Comment (willclarktech) [reply packages/tendermint-rpc/src/client.spec.ts:143]: But pubkey does!

File: packages/tendermint-rpc/src/client.spec.ts [hunk 2]
```diff
@@ -130,14 +130,31 @@ function defaultTestSuite(rpcFactory: () => RpcClient, adaptor: Adaptor, expecte
     client.disconnect();
   });
 
+  it("can get validators", async () => {
```
Comment (ethanfrey) [inline packages/tendermint-rpc/src/client.spec.ts:133]: nice

**Review by ethanfrey — APPROVED** [review]
Looking good

**Pull request merged**